multi-node/aws/pkg/cluster/cluster.go

```diff
@@ -254,3 +263,22 @@ func (c *Cluster) Destroy() error {
 	_, err := cfSvc.DeleteStack(dreq)
 	return err
 }
+
+func (c *Cluster) validateKey() error {
```

Comment:
Check out how `validateExistingVPCState` is [set up](`validateExistingVPCState` takes an ec2